.github/workflows/build.yml

```diff
@@ -0,0 +1,34 @@
+name: Build Docker images
+
+on:
+  pull_request:
+  push:
```

Comment:
Does this mean again double execution of all tasks (once at push, then again at PR)?

Reply:
Yes, unfortunately. :( I would be happy to adopt @praveksharma's suggestion for [reducing this](https://github.com/open-quantum-safe/liboqs/pull/1902#issuecomment-2310264472), though.

As an aside, I looked into the duplicate run issue [a while time ago](https://github.com/open-quantum-safe/tsc/issues/5#issuecomment-2029868313) and didn't find any automated solution that I was happy with.

Reply:
you could limit the push to main branch

Reply:
OK here -- this stuff doesn't run as often as code changes in `liboqs`...